### ISSUE TITLE
Adding a Powershell example

### ISSUE THREAD

**Issue by mavmedia** [ryanluker/vscode-coverage-gutters#456]
**Is your feature request related to a problem? Please describe.**
No, but we are currently working a lot with powershell and I would like to add a powershell / pester example.

**Describe the solution you'd like**
I already have a working example and would like to contribute by adding it to the solution.

**Describe alternatives you've considered**
None.

**Additional context**
This is my first ever attempt at contributing on github, so further hints beside of [CONTRIBUTING.md](https://github.com/ryanluker/vscode-coverage-gutters/blob/master/CONTRIBUTING.md) are much appreciated.

**Comment by ryanluker** [comment]
@mavmedia Thanks for the issue!

We always welcome new examples (we try to add them to the test suite as well, to confirm everything is still working for that example).

Here is the last example that was added, it should give you a good starting place (thanks for checking on the contributing README already as well). I also linked some of the PRs that also added tests to the test suite.

https://github.com/ryanluker/vscode-coverage-gutters/pull/405
https://github.com/ryanluker/vscode-coverage-gutters/pull/441
